clients/cobol-lsp-vscode-extension/src/__tests__/services/BridgeForGitSupport.spec.ts

```diff
@@ -74,10 +74,10 @@ const pgMapJson = { pgms: [{ program: "main.cob", pgroup: "pg1" }] };
 
 describe("Bridge for Git group tests", () => {
   test("Decode b4f json", () => {
-    const result = decodeBridgeJson(b4gJson);
+    const result = b4gJson;
     expect(result).toBeDefined();
-    expect(result!.elements["main"].processorGroup).toBe("pg2");
-    expect(Object.keys(result!.elements)[0] + "." + result!.fileExtension).toBe(
+    expect(result.elements["main"].processorGroup).toBe("pg2");
+    expect(Object.keys(result.elements)[0] + "." + result.fileExtension).toBe(
       "main.cob",
     );
   });
```

Comment:
I think we can just drop this test... It was supposed to check the `decodeBridgeJson` validation. In the current form it does not make much sense to keep it.

Reply:
updated